stdlib/2/gettext.pyi

```diff
@@ -24,20 +24,20 @@ class NullTranslations(object):
     def ngettext(self, singular: str, plural: str, n: int) -> str: ...
     def lngettext(self, singular: str, plural: str, n: int) -> str: ...
     def ungettext(self, singular: Union[str, unicode], plural: Union[str, unicode], n: int) -> unicode: ...
-    def info(self) -> Any: ...
-    def charset(self) -> Any: ...
-    def output_charset(self) -> Any: ...
-    def set_output_charset(self, charset: Any) -> None: ...
-    def install(self, unicode: bool = ..., names: Any = ...) -> None: ...
+    def info(self) -> Dict[str, str]: ...
+    def charset(self) -> Optional[str]: ...
+    def output_charset(self) -> Optional[str]: ...
+    def set_output_charset(self, charset: Optional[str]) -> None: ...
+    def install(self, unicode: bool = ..., names: Container[str] = ...) -> None: ...
 
 class GNUTranslations(NullTranslations):
     LE_MAGIC = ...  # type: int
     BE_MAGIC = ...  # type: int
 
-def find(domain: str, localedir: str = ..., languages: List[str] = ...,
+def find(domain: str, localedir: Optional[str] = ..., languages: Optional[Sequence[str]] = ...,
          all: Any = ...) -> Optional[Union[str, List[str]]]: ...
 
-def translation(domain: str, localedir: str = ..., languages: List[str] = ...,
-                class_: Any = ..., fallback: Any = ..., codeset: Any = ...) -> NullTranslations: ...
-def install(domain: str, localedir: str = ..., unicode: Any = ..., codeset: Any = ...,
-            names: Any = ...) -> None: ...
+def translation(domain: str, localedir: Optional[str] = ..., languages: Optional[Sequence[str]] = ...,
+                class_: Optional[type] = ..., fallback: Any = ..., codeset: Optional[str] = ...) -> NullTranslations: ...
```

Comment:
Should this be `Type[NullTranslations]]`?

Also, it looks like `fallback` is a bool.

Reply:
Both fixed and one more for `unicoide`